### DISCUSSION TITLE
add flashrom support for sharp LH28F016SCT-L95

### DISCUSSION THREAD

**Discussion by kazmat6502** [sanni/cartreader#306] [Coding Corner]
Hello Sanni
I would like to add sharp LH28F016SCT-L95 flashrom for SNES repro cart.

datsheet is here:  https://drive.google.com/file/d/1atHdXkYCGmuvQiJua6jYSv0oN2rz9-S0/view?usp=sharing

**Comment by sanni** [comment]
I have stopped adding new flashroms, not worth my time, but if you have a specific question I might be able to give you some advice.
For the beginning you should read through both flash.ino and the datasheet, copy&paste some functions together and then report back if you get stuck somewhere.
